docs/content/modeling/user-groups.mdx

```diff
@@ -175,31 +174,31 @@ In order to assign `member`s of a `team` a relation to a `document`, we can crea
   ]}
 />
 
-### 04. Checking An Individual Member's Access To An Object
+### 04. Check An Individual Member's Access To An Object
 
-Now that we have:
+Now that you have:
 
-- a relationship tuple indicating that **alice** is an `member` of **team:writers**
-- a relationship tuple indicating that **members of team:writers are editors of document:meeting_notes.doc**
+- a relationship tuple indicating that `alice` is an `member` of `team:writers`
+- a relationship tuple indicating that `members` of `team:writers` are editors of `document:meeting_notes.doc`
```

Comment:
`members` everywhere else is styled as `member`s -- i prefer the former, but remain consistent throughout.